better-politics-mod/common/ideologies/bpm_leader_ideologies.txt

```diff
@@ -736,6 +774,14 @@ ideology_reformer = {
 		law_debt_slavery = strongly_disapprove
 		law_slave_trade = strongly_disapprove
 	}
+
+	lawgroup_repeatable_culture = {
```

Comment:
I think adding stances on political repeatables for reformers will be worthwhile too